packages/manager/apps/web/client/app/hosting/database/order/components/private-database/private-database.html

```diff
@@ -0,0 +1,176 @@
+<div class="text-center" data-ng-if="$ctrl.loadingDependencies">
+    <oui-spinner data-size="l"></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loadingDependencies">
+    <p data-translate="hosting_database_order_private_description"></p>
+
+    <oui-stepper
```

Comment:
`product-offers` component could be used here, I see configuration steps that could be transcluded, and duration/validation step are quite the same than in the component

Reply:
I got what you are saying. I did not write this code. I just extracted the existing code into a separate reusable component so that it can be used in 2 places. Let me check if I can use the product-offers component.